Library/Formula/procyon-decompiler.rb

```diff
@@ -0,0 +1,27 @@
+class ProcyonDecompiler < Formula
+  desc "Procyon Java Decompiler is part of the Procyon suite of Java metaprogramming tools focused on code generation and analysis."
```

Comment:
Got it, and apparently the name is included in the calculation, so i'll make another attempt.